includes/class-wc-payments.php

```diff
@@ -1325,6 +1325,26 @@ public static function get_order_service(): WC_Payments_Order_Service {
 		return self::$order_service;
 	}
 
+	/**
+	 * Returns the token service instance.
+	 *
+	 * @return WC_Payments_Token_Service
+	 */
+	public static function get_token_service(): WC_Payments_Token_Service {
+		return self::$token_service;
```

Comment:
Declare this private static property.

Reply:
@gpressutto5 Sorry, I'm not sure what you mean here...

`token_service` is already a private static property on this class (line 97)... and if you mean making this function private, I'm confused. Surely that would prevent the rest-payments-orders-controller from using it, which is why we added it in the first place? The other similar getters are all public functions too.

Excuse my ignorance if I'm missing something obvious... I'm here to learn 😄

Reply:
You're right, sorry I missed that!